test_integration/03_stake/01_stake_helper.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const { assert } = require('chai');
+const BN = require('bn.js');
+const StakeHelper = require('../../src/helpers/StakeHelper');
+const shared = require('../shared');
+
+const validateReceipt = (receipt) => {
```

Comment:
We are having the common functions in all the integration test files.
Maybe we can move it to a common file. (We can create a new ticket for this)

Reply:
I will create a ticket. This could potentially go into a separate utils repo or something like that.

Reply:
https://github.com/OpenSTFoundation/mosaic.js/issues/98